### ISSUE TITLE
Add tdm compatibility for nuketown

### ISSUE THREAD

**Issue by jextrevor** [kinvaris/openspades-maps#1]
The nuketown has a (known) issue where spawning on non-arena mode puts you into the water. Arena mode works fine. This ought to be noted in the comments.

I'm working on a version of nuketown.txt that supports tdm play.

**Comment by kinvaris** [comment]
Indeed, this is correct! You can always make a pull request to improve the current maps (or add some)
